sphinx/config.py

```diff
@@ -209,6 +209,7 @@ class Config(object):
                            None),
         latex_logo = (None, None, string_classes),
         latex_appendices = ([], None),
+        latex_macros_onlywithprefix = (False, None),
```

Comment:
indeed I have had big troubles with finding a name. The commit log explains that it applies only currently to the text styling macros. Your suggested names (which I agree are better) are opposite boolean, default value True. Perhaps `latex_keep_oldmacronames` ?